omnibus/.kitchen.yml

```diff
@@ -11,7 +11,9 @@ driver:
     cpus: 4
     memory: 4096
   synced_folders:
-    - ['../..', '/home/vagrant']
```

Comment:
What's this about? Having it linked in this way makes it possible to actually build on vagrant without having to copy the home dir first--changing it back to linking chefdk directly is going to cause it to fail again.